ddtrace/tracer/tracer.go

```diff
@@ -8,6 +8,8 @@ package tracer
 import (
 	gocontext "context"
 	"encoding/binary"
+	"fmt"
+	"github.com/google/uuid"
```

Comment:
This should go to the second group. Or try running https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
addressed in 7bb75db8922561f592e62fa1c7bd05bc5b442903